addons/web/static/tests/webclient/actions/embedded_action.test.js

```diff
@@ -178,9 +177,10 @@ defineActions([
 ]);
 
 onRpc("set_res_users_settings", ({ kwargs }) => ({
-    id: kwargs.context.uid,
+    id: 1,
```

Comment:
those two mocks do not seem very robust as they return what was already there in the first place